packages/ui/src/pages/Register/index.tsx

```diff
@@ -35,6 +29,10 @@ const Register = () => {
     return <CreateAccount />;
   }, [method]);
 
+  if (!['email', 'sms', 'username'].includes(method)) {
```

Comment:
better have an enum or type for this?

Reply:
Actually, there is a sign-in method type defined for those components use only.  As for this one, as the 'method' come from the route params, strickly it could be any string. So have to treat it as any string type, instead guard the input type at the inner component level.